offload/liboffload/API/Program.td

```diff
@@ -24,6 +24,18 @@ def olCreateProgram : Function {
     let returns = [];
 }
 
+def olIsValidBinary : Function {
```

Comment:
I'd prefer `olIsCompatibleBinary`; an ELF file (for example) can be a perfectly valid file yet not be compatible with the device.